### ISSUE TITLE
What are the requirements for using one's own dataset?

### ISSUE THREAD

**Issue by zhangjy328** [A03ki/f-AnoGAN#21]
Does this model have any requirements for the size of the input images? Can it be used for large-sized images?

**Comment by zhangjy328** [comment]
I want to train an image dataset of rectangular shapes, but I don't want to perform any size changes. Is that okay?

**Comment by A03ki** [comment]
Hi, @zhangjy328.

The images will be resized to the image_size specified as an argument (default is 64×64). In other words, if the images are rectangular, they will be converted to square.

**Comment by zhangjy328** [comment]
And, Do I have to train only one type of sample each time? I tried to include multiple types of samples in the training set simultaneously, but the loss value never converged, and the generation results were very poor. The total number of samples is 10,000.
![企业微信截图_17344851897569](https://github.com/user-attachments/assets/ff351c66-4ebd-4f5d-b780-930bc2dc3fe2)
